Public/Src/Tools/UnitTests/Analyzers/PipExecutionPerformanceAnalyzerTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using BuildXL.Execution.Analyzer;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Test.Tool.Analyzers
+{
+    public class PipExecutionPerformanceAnalyzerTests : AnalyzerTestBase
+    {
+        public PipExecutionPerformanceAnalyzerTests(ITestOutputHelper output) : base(output)
+        {
+           
+        }
+
+        [Fact]
+        public void QuoteEscapeValidation()
+        {
+            char quote = '"';
+            char slash = '\\';
+            Assert.True(("" + slash + slash).Equals(PipExecutionPerformanceAnalyzer.NormalizeString("" + slash)));

```

Comment:
Instead of `Assert.True(expected.Equals(actual))` use `XAssert.AreEqual(expected, actual)` because the latter will produce a more informative error message when the assertion fails